design/ui/pagination/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@espressive/pagination",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "es/index.js",
+  "scripts": {
+    "rollup": "rm -rf dist && rm -rf es && BABEL_ENV=production rollup -c",
+    "rollup:watch": "rm -rf dist && rm -rf es && BABEL_ENV=production rollup -c --watch"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
```

Comment:
These will need to get changed this week to 'espressive' instead of 'react-app' now that we have our own. 🙌